src/act.core.web/Services/BuildSpecificationFactory.cs

```diff
@@ -393,6 +393,7 @@ public async Task<BuildSpec> BuildSpecification(long specId)
             var appSpec = await _ctx.BuildSpecifications.AsNoTracking()
                 .Include(p => p.Parent)
                 .Include(p => p.Nodes)
+                .Include(p=>p.Owner)
```

Comment:
need to add some spaces here for readability.

Reply:
fixed